galsim/galaxy_sample.py

```diff
@@ -137,7 +121,9 @@ class COSMOSCatalog:
                             - "marginal": Apply the above cuts, plus ones that eliminate some more
                               marginal cases.
 
-                            [default: "marginal"]
+                            Use of "bad_stamp" or "marginal" requires a ``CATALOG_selection.fits``
+                            file (where CATALOG refers to the regular catalog base name).
```

Comment:
Say how/where that base name is specified?

Reply:
Changed to "(where CATALOG is ``file_name`` without the ".fits" extension)".